guides/v15-mdc-migration.md

```diff
@@ -0,0 +1,571 @@
+  not specified. All newer appearance settings require explicitly specifying a `<mat-label>` if one
+  was not provided before. This change addresses an accessibility best practice of not using labels
+  and placeholders interchangeably.
+
+* By default, MatFormField still reserves exactly one line of space below the field for hint or
+  error text. The new subscriptSizing setting (available as an Input and via the
+  MAT_FORM_FIELD_DEFAUlT_OPTIONS injectable) allows changing this behavior to always match the size
+  of the text.
+
+* The text inside `<mat-hint>` is larger and darker in order to meet W3C text guidelines.
+
+* While the previous form-field had a single directive for prefixes (`matPrefix`) and a single 
+  directive for suffixes (`matSuffix`), the MDC-based form-field distinguishes between text
+  prefix/suffixes which are baseline aligned with the input text, and icon prefix/suffixes which are
+  center aligned in the form-field. Use `matTextPrefix` or `matTextSuffix` to indicate a text
+  prefix/suffix, and `matIconPrefix` or `matIconSuffix` to indicate an icon prefix/suffix. The old
+  `matSuffix` and `matPrefix` APIs will behave like icons, though they are now deprecated.
+
+* The MDC-based form-field has a new `@Input() subscriptSizing: 'fixed'|'dynamic'`. When this
+  setting is set to `fixed` (default), the form-field reserves enough space in the layout to show
+  one line of hint or error text. When set to `dynamic`, the form-field expands and contracts the 
+  amount of space it takes in the layout to fit the error / hint that is currently shown.
+
+### Input
+
+* MatInput must be inside `<mat-form-field>`. Previously it was (unintentionally) possible to use an
+  `<input matInput>` without the form field if the page loaded form field styles elsewhere.
+
+* The MDC-based MatInput hides the native calendar picker indicator associated with
+  `<input matInput type="date">`, if you want this indicator to appear for your inputs, use the
+  following CSS:
+
+    ```scss
+    .mat-mdc-input-element::-webkit-calendar-picker-indicator {
+      display: block;
+    }
+    ```
+
+### List
+
+* The API has been reworked to support text wrapping and better integration with the Material Design
+  specification.
+
+* Previously, list items were commonly created using multiple `span` elements with the `matLine`
+  directive applied. Each `span` resulting in a line, and the first one becoming the primary line.
+  With the new API, the `matLine` directive has been split into two more granular and obvious
+  directives:
+  * `matListItemTitle`
+  * `matListItemLine`
+
+* Every list item is supposed to have a title for accessibility. If no explicit title and lines are
+  provided, the direct content of the `<mat-list-item>` will become the title. You cannot have
+  multiple line directives without a title.
+
+  ```html
+  <mat-list-item>This is the title</mat-list-item>
+  ```
```

Comment:
I'm confused by this one. We don't apply any a11y pattern for `<mat-list>` + `<mat-list-item>`, so I'm not sure what this is talking about. Even it was a plain `role="list"`, the entire contents of the list item would be its label in the a11y tree

Reply:
This is c/p from internal docs, it looks like @devversion is the original author - can you clarify?

Reply:
We can just remove the accessibility portion. I don't fully recall where this came from and I think Jeremy is right. A title is still required- but mostly for the visual requirements/specification of the Material Design component.